### DISCUSSION TITLE
Which takes priority - Trakt or Plex?

### DISCUSSION THREAD

**Discussion by davidbennett1979** [Taxel/PlexTraktSync#1848] [Q&A] [answered]
If I have existing entries in Trakt (from another Plex server) and I point plextraktsync at a new Plex server with differing data, which data does Plextraktsync select?

Does it update Trakt with the new Plex servers data?
or
Does it update the new Plex server with the existing Trakt data I sync'd from the original Plex server?

Said another way, is there a way to force from Trakt to Plex only?

Thanks!

**Comment by simonc56** [comment]
What data are you talking about ?

**Reply by davidbennett1979** [reply]
Watched and progress.

I have trakt updated with watch stats from a Plex database with Movies and TV.

I am now going to point plextraktsync at a new Plex Database with the same movies and TV and a different watch history.

Which will get over written? Trakt or the new Plex Database?

**Reply by simonc56** [reply, marked as answer]
`unwatched` is overwritten by `watched`.

**Comment by davidbennett1979** [comment]
thank you so much very helpful

**Reply by glensc** [reply]
reply in thread and mark acceptable post as answer

**Comment by glensc** [comment]
`sync` command has `--dry-run` option. use it to see what changes will be made without actually making the changes.

additionally you can disable `plex_to_trakt` or `trakt_to_plex` sync config that you do not want to sybc
